src/Microsoft.Security.Utilities.Core/IdentifiableSecrets.cs

```diff
@@ -995,7 +1000,18 @@ internal static byte[] ConvertFromBase64String(string text)
     {
         text = TransformToStandardEncoding(text);
         text += RetrievePaddingForBase64EncodedText(text);
-        return Convert.FromBase64String(text);
+        byte[] result;
+
+        try
+        {
+            result = Convert.FromBase64String(text);
+        }
+        catch
```

Comment:
```suggestion
        catch (FormatException)
```

It's not as bad when general catch is scoped like this, but even here, swallowing ArgumentNullException could hide a bug.

Reply:
That never should have been in there, sigh.

Reply:
OK, turns out my final commit wasn't pushed. Since I didn't literally get the change out in a coherent form, I give up, I will go back and address the test comments. :)